WebWork

Marine10 opened the pull request.

rosa commented:
Hi @Marine10, thanks for sending this pull request! I had a look at the URL listed for this integration and the logo used belongs to Basecamp 2. Is this integration for Basecamp 3 or 2? 
Thank you!

Marine10 commented:
Hi,

Thank you very much for checking the request.
It is Integration for Basecamp 3. It seems that our team members put the
wrong logo.

If you could send the right one, we will change it immediately.

Best regards,
Marine


On Fri, Jan 24, 2020 at 2:30 AM Rosa Gutierrez <notifications@github.com>
wrote:

> Hi @Marine10 <https://github.com/Marine10>, thanks for sending this pull
> request! I had a look at the URL listed for this integration and the logo
> used belongs to Basecamp 2. Is this integration for Basecamp 3 or 2?
> Thank you!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/basecamp/bc3-integrations/pull/144?email_source=notifications&email_token=AMHSH5C5DSB2EOCXV6DQK43Q7ILBLA5CNFSM4KKEZSYKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJZDAKI#issuecomment-577908777>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMHSH5AZJKRU22YGVODGFMDQ7ILBLANCNFSM4KKEZSYA>
> .
>

rosa commented:
Thank you @Marine10! You can find the logo [here](https://basecamp.com/about/press), under the "Download the Basecamp Logo" section.

Marine10 commented:
Hi Rosa,

We have changed the logo if it is still showing the old one, just hard
refresh the page (Ctrl+Shift+R).

Please, let me know if there is anything else we can do.

Best regards,
Marine

On Fri, Jan 24, 2020 at 12:31 PM Rosa Gutierrez <notifications@github.com>
wrote:

> Thank you @Marine10 <https://github.com/Marine10>! You can find the logo
> here <https://basecamp.com/about/press>, under the "Download the Basecamp
> Logo" section.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/basecamp/bc3-integrations/pull/144?email_source=notifications&email_token=AMHSH5HLNIUGHNWJQ7PHYADQ7KRO5A5CNFSM4KKEZSYKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJ2CUGA#issuecomment-578038296>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMHSH5CGVUHUTMG2LO53TN3Q7KRO5ANCNFSM4KKEZSYA>
> .
>

rosa commented on integrations.yml:
```diff
@@ -390,6 +390,12 @@
+  website: "https://www.webwork-tracker.com/integrations/basecamp"
+  icon: "webwork.png"
+  category: 2
 -
```
Could you move the integration to be at the end of the file, under the `Centreli` one?

rosa commented:
Thank you! That's perfect. I just added a small comment, when that's resolved I'll merge this.

Marine10 commented:
Thanks.
Can you, please, check it? I have just removed it.

Thank you in advance!



On Fri, Jan 24, 2020 at 4:00 PM Rosa Gutierrez <notifications@github.com>
wrote:

> Thank you! That's perfect. I just added a small comment, when that's
> resolved I'll merge this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/basecamp/bc3-integrations/pull/144?email_source=notifications&email_token=AMHSH5E44KJLQ7ZDNDNDIGLQ7LJ6JA5CNFSM4KKEZSYKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJ2SM3Y#issuecomment-578102895>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMHSH5FCWJNYGYFZKAOKXLTQ7LJ6JANCNFSM4KKEZSYA>
> .
>
